### PR TITLE
lightning, importinto: reduce disk quota interval

### PR DESCRIPTION
**Pull request by GMHDBJD**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/44926

Problem Summary:

### What is changed and how it works?
reduce diskquota check interval, from 1m -> 10s
### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by niubell — APPROVED** [review]
LGTM

File: br/pkg/lightning/config/config.go [hunk 1]
```diff
@@ -937,7 +937,7 @@ func NewConfig() *Config {
 		Cron: Cron{
 			SwitchMode:     Duration{Duration: DefaultSwitchTiKVModeInterval},
 			LogProgress:    Duration{Duration: 5 * time.Minute},
-			CheckDiskQuota: Duration{Duration: 1 * time.Minute},
+			CheckDiskQuota: Duration{Duration: 10 * time.Second},
```
Comment (D3Hunter) [inline br/pkg/lightning/config/config.go:940]: we don't need to change this, user can config on their need, we have doc about it

https://docs.pingcap.com/tidb/dev/tidb-lightning-physical-import-mode-usage

File: executor/importer/table_import.go [hunk 2]
```diff
@@ -54,7 +54,7 @@ var NewTiKVModeSwitcher = local.NewTiKVModeSwitcher
 var (
 	// CheckDiskQuotaInterval is the default time interval to check disk quota.
 	// TODO: make it dynamically adjusting according to the speed of import and the disk size.
-	CheckDiskQuotaInterval = time.Minute
+	CheckDiskQuotaInterval = 10 * time.Second
```
Comment (D3Hunter) [inline executor/importer/table_import.go:57]: would be better to have a configurable value as comment says

**Review by D3Hunter — APPROVED** [review]

**Pull request merged**